concepticondata/conceptlists/Epps-2021-843.tsv

```diff
@@ -841,4 +841,4 @@ Epps-2021-843-839	839	YARD OR COURT	YARD OR COURT 1 // YARD OR COURT 2	736	YARD
 Epps-2021-843-840	840	YEAR	YEAR	1226	YEAR	år	2293	år	2819	c	0	1	1
 Epps-2021-843-841	841	YOKE	YOKE 1 // YOKE 2 // YOKE 3	14	YOKE	børetre	0				0	0	0
 Epps-2021-843-842	842	YOLK	YOLK	1556	YOLK	eggegule		äggula	3	c	0	0	0
-Epps-2021-843-843	843	YOUTH	YOUTH			ungdom		ungdom		c	0	1	1
+Epps-2021-843-843	843	YOUTH	YOUTH	1304	CHILD (YOUNG HUMAN)	ungdom		ungdom		c	0	1	1
```

Comment:
The swedish gloss points to an abstract concept, like German Jugend. So mapping is not justified in my opinion.